tests/MageTest/Command/AbstractCommandTest.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+namespace MageTest\Command;
+use Mage\Command\AbstractCommand;
+use MageTest\TestHelper\BaseTest;
+use PHPUnit_Framework_MockObject_MockObject;
+
+/**
+ * Class AbstractCommandTest
+ * @package MageTest\Command
+ * @author Jakub Turek <ja@kubaturek.pl>
+ * @coversDefaultClass Mage\Command\AbstractCommand
+ */
+class AbstractCommandTest extends BaseTest
+{
+    /**
+     * @var AbstractCommand|PHPUnit_Framework_MockObject_MockObject
+     */
+    private $abstractCommand;
+
+    /**
+     * @before
+     */
+    public function before()
+    {
+        $this->abstractCommand = $this->getMockForAbstractClass('Mage\Command\AbstractCommand');
+    }
+
+    /**
+     * @covers ::setConfig
+     */
+    public function testSetConfig()
+    {
+        $configMock = $this->getMock('Mage\Config');
+        $this->abstractCommand->setConfig($configMock);
+
+        $actual = $this->getPropertyValue($this->abstractCommand, 'config');
+        $this->assertEquals($configMock, $actual);
+    }
+
+    /**
+     * @covers ::getConfig
+     */
+    public function testGetConfig()
```

Comment:
Umm ok. Sorry for my yesterday note, I was thinking of dependency injection instead of depend-annotation (I was exhausted, don't ask why).
I see advantages of such solution, but some testing philosophies say that tests should be completely independent. In the other hand these are just setters, the methods with path complexity equal 1. So well.. I need to consider it. But thanks for spotting! ;)